appveyor.yml

```diff
@@ -23,5 +30,4 @@ install:
 build: false
 
 test_script:
-  - cd c:\projects\habitat\components\hab
-  - cargo build --verbose
+  - appveyor.bat
```

Comment:
Would be great to see this in powershell because `%%A` makes me reactively stab my eyes :)